Documentation/HowToGuides/AddingACollisionIgnorer/README.md

```diff
@@ -0,0 +1,64 @@
+# Adding A Collision Ignorer
+
+> * Level: Beginner
+>
+> * Reading Time: 5 minutes
+>
+> * Checked with: Unity 2018.3.14f1
+
+## Introduction
+
+The `Input Mutator ColliisonIgnorer` provides a way of allowing two or more objects to not interact with each other.
+
+## Let's Start
+
+### Step 1
+
+Create a new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 0`, `Z = 0`
+
+![Adding A Cube](assets/images/AddingACube.png)
+
+### Step 2
+
+Create another new `Cube` Unity 3D Object by selecting `Main Menu -> GameObject -> 3D Object -> Cube` and change the `Transform` properties to:
+
+* Position: `X = 0`, `Y = 2`, `Z = 0`
+
+![Adding The Second Cube](assets/images/AddingTheSecondCube.png)
+
+### Step 3
+
+Click the `Add Component` button and select the `Rigidbody` component to the second `Cube`. 
+
+![Adding A Rigidbody](assets/images/AddingARigidbody.png)
+
+Play the `Unity Scene` and you should see the second `Cube` land on top of the first `Cube`.
+
+![Playing The Unity Scene With The Rigidbody](assets/images/PlayingTheUnitySceneWithTheRigidbody.png)
+
+### Step 4
+
+Expand the `Tilia Input Mutators CollisionIgnorer` Package directory in the Unity Project window and select then `Packages -> Tilia Input Mutators CollisionIgnorer -> Runtime -> Prefabs -> Actions` directory then drag and drop the `Input.Mutators.ColliisonIgnorer` prefab into the Hierarchy window.
+
+![Drag And Drop Collision Ignorer Into Hierachy](assets/images/DragAndDropCollisionIgnorerIntoHierachy.png)
+
+
+### Step 5
+
+Under `Sources` in the `Rigidbody` settings change the `size` value to `1` and drag and drop the first `Cube` into the box displaying the name `Element 0`.
+
+![Drag And Drop Cube Into Element](assets/images/DragAndDropCubeIntoElement.png)
+
+### Step 6
+
+Under `Targets` in the `Rigidbody` settings change the `size` value to `1` and drag and drop the second `Cube` into the box displaying the name `Element 0`.
+
+![Drag And Drop Second Cube Into Element](assets/images/DragAndDropSecondCubeIntoElement.png)
+
+Play the `Unity Scene` you should see the `Cube` fall right through the second `Cube`.
```

Comment:
Use the GameObject names for cube so: (also unity scene doesn't need to be in backticks)

Play the Unity scene you should see `Cube (1)` fall right through `Cube` this is because the collisions between the two GameObjects is now being ignored.